flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/plan/nodes/CommonCorrelate.scala

```diff
@@ -46,7 +47,7 @@ trait CommonCorrelate {
     inputSchema: RowSchema,
     udtfTypeInfo: TypeInformation[Any],
     returnSchema: RowSchema,
-    joinType: SemiJoinType,
+    joinType: JoinRelType,
```

Comment:
I notice that this change added a number of new warnings regarding that the new `ANIT` and `SEMI` joins are not handled in some of the switch case clause across the table planner. Are we planning to have a follow up JIRA to fix this?

Reply:
sorry for late reply, I create a JIRA to fix this: https://issues.apache.org/jira/browse/FLINK-13347